docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+# Backup NoSQL database guide
```

Comment:
```suggestion
# A Guide on Taking Backups of NoSQL Databases on a Kubernetes Environment
```

We might need some naming rules (e.g., capitalization, strucfor this kind of guide.
@josh-wong Thought?

Reply:
Regarding title and heading capitalization, I'd like us to follow similar capitalization guidelines mentioned in the [Microsoft Style Guide ("Formatting headings")](https://learn.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings) and [Google Developer style guide ("Headings and title format")](https://developers.google.com/style/headings#heading-and-title-format). Both style guides recommend using sentence-style capitalization.

Regarding title structure, I'd like us to follow a similar style mentioned in the [Google Developer style guide ("Heading and title text")](https://developers.google.com/style/headings#heading-and-title-text). This style guide shows how to write tasked-based and non-task-based headings.

I'll be sure to add both of the above to our style guide.

Reply:
Following the guidance in [Google Developer Documentation Style Guide ("Heading and title text")](https://developers.google.com/style/headings#heading-and-title-text), how about we use the following?:

```suggestion
# Back up NoSQL databases in a Kubernetes environment
```